### DISCUSSION TITLE
Looking for Patch for MHI2_ER_AU43x_P5402 MU1467 (TT) or newer FW Version: MU1479 or MU1489

### DISCUSSION THREAD

**Discussion by Jay-T88** [Mr-MIBonk/M.I.B._More-Incredible-Bash#208] [General]
Dear Team,
unfortunately my version of MMI is not supported yet... "MHI2_ER_AU43x_P5402 MU1467"
There are patches for these versions only: "MHI2_ER_AU43x_P5404_MU1479" or "MHI2_ER_AU43x_P5406_MU1489"
Can you please either make a patch for my version "P5402 MU1467" or alternatively can someone please provide the update firmware to MU1479 or MU1489 - then I can apply the existing patches myself.

thanks a lot

BTW: Does somebody have a link to change log of the different FW versions MU1467 -> MU1479 / MU1489?

**Comment by harman-f** [comment]
Those FW are not available to us.
Share a copy of your backup.

**Comment by Jay-T88** [comment]
I have uploaded "MHI2_ER_AU43x_P5402 MU1467" to mibsolutions M.I.B_Backup Folder: https://mibsolution.one/index.php/f/181591

**Reply by harman-f** [reply]
check M.I.B folder on mibsolution

**Reply by Jay-T88** [reply]
Thx a lot!!! Patch seemed to work as I was able to patch FEC to 23000EE and install latest maps - excellent!
Interestingly SVM Fix seems not to work correctly as I still get an error message, but MMI works great so no big deal yet.
Will do some more testing... Thanks again for the great support!!!

**Reply by G15V** [reply]
> check M.I.B folder on mibsolution

Do you have a donwload link for this file as mibsolution is currently down?

Best regards,
Benjamin

**Reply by leesilverconsultancy-debug** [reply]
Hi, following on from this thread — I'm on MHI2_ER_AU43X_P4161_1 (MU1133) trying to reach P5406 (MU1489) to apply the M.I.B. patch, but the SWDL process reverts/rejects during verification using files I sourced from VAG_FlashFiles (searched "1489"). Does anyone have a confirmed-working direct upload of P5406_MU1489 in the M.I.B_Backup folder, similar to what was shared above for P5402_MU1467? Or can confirm the correct file set/folder structure for a from-scratch SWDL firmware update on this train? Thanks.

**Comment by YmeGT** [comment]
Good day, 

How do I download from the MIB Backup folder?
